### PR TITLE
Update codecov.yml to fit configure format change

### PR DESCRIPTION
**Pull request by junw-33455432**

### CODE REVIEW

**Review by nannany00 — APPROVED** [review]

File: codecov.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  precision: 2        
+  round: down         
+  range: 20...80     
+  status:
+    project:                   
+      default:                 
+        enabled: yes           
+        target: 80%                 
+        if_no_uploads: error   
+        if_not_found: error
```
Comment (surite) [inline codecov.yml:11]: option  should be one of 'success' and 'failure', refer to https://docs.codecov.com/docs/commit-status#if_not_found

File: codecov.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        target: 80%                 
+        if_no_uploads: error   
+        if_not_found: error  
+        if_ci_failed: error   
+    patch:
+      default:
+        enabled: true            
+        target: 80%                      
+        if_no_uploads: error     
+        if_not_found: error
```
Comment (surite) [inline codecov.yml:18]: same with above

File: codecov.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+coverage:
+  precision: 2        
+  round: down         
+  range: 20...80
```
Comment (surite) [inline codecov.yml:4]: if the target is 80%, why the green range is less than 80%?

File: codecov.yml (hunk 1 above)
Comment (junw-33455432) [reply codecov.yml]: Thanks for the comment, it has been updated.

File: codecov.yml (hunk 2 above)
Comment (junw-33455432) [reply codecov.yml]: Updated as above one.

**Review by surite — APPROVED** [review]

**Review by renligeng — APPROVED** [review]

**Review by renligeng — APPROVED** [review]

**Pull request merged**